src/libsat355.cpp

```diff
@@ -0,0 +1,208 @@
+// self
+#include "libsat355.h"
+
+// std
+
+#include <iostream>
+
+// orbittools
+// *NOTE: Large portions of this was stolen from orbittools demo
+// "coreLib.h" includes basic types from the core library,
+// such as cSite, cJulian, etc. The header file also contains a
+// "using namespace" statement for Zeptomoby::OrbitTools.
+#include "orbitTools/core/coreLib.h"
+// "orbitLib.h" includes basic types from the orbit library,
+// including cOrbit.
+#include "orbitTools/orbit/orbitLib.h"
+
+// IOS Core Foundation: Date::init(timeIntervalSinceReferenceDate: TimeInterval)
+const double EPOCH_JAN1_00H_2001 = 2451910.5; // Jan  1.0 2001 = Jan  1 2001 00h UTC
+
+// TRICKY: extern "C"- Make functions callable from SwiftUI.
+// Force orbit_to_lla() to be "C" rather than "C++" function.
+// Needed because SwiftUI binding header can only call into "C".
+
+extern "C" {
+
+int HelloWorld()
+{
+	std::cout << "Hello World!" << std::endl;
+
+	return kOK;
+}
+
+// orbit_to_lla:
+// Calculate satellite Lat/Lon/Alt for time "now" using
+// input TLE-format orbital data
+int orbit_to_lla(	const char* in_tle1,	// TLE (Sat Name)
+					const char* in_tle2,	// TLE line 1
+					const char* in_tle3,	// TLE line 2
+					double* out_tleage,		// age of TLE in secs since: Jan 1, 2001 00h UTC
+					double* out_latdegs,	// latitude in degs
+					double* out_londegs,	// longitude in degs
+					double* out_altkm,
+					std::time_t inTime)		// altitude in km
```

Comment:
A "dll API" is always C-based API.
Meaning it can only use C-compatible data types.
`std::time_t` is C++ only.
We need a way to pass a "time" in a portable manner